searchmenu/init.lua

```diff
@@ -14,6 +14,15 @@ local add_binds, add_cmds = modes.add_binds, modes.add_cmds
 local menu_binds = binds.menu_binds
 local new_mode = require("modes").new_mode
 
+local function try_search_engine (url, selection)
+    local ret
+    if type(url) == "string" then
+        ret = string.format(url, luakit.uri_encode(selection))
```

Comment:
Is there any reason not to just `return ...` inside the if statements? doesn't look like multiple return values would be an issue